js/script.js

```diff
@@ -0,0 +1,15 @@
+const navSlide = () => {
+  const burger = document.querySelector('.humberger-menu');
+  const nav = document.querySelector('.desktop-nav');
+  burger.addEventListener('click', () => {
+    nav.classList.toggle('nav-active');
+    burger.classList.toggle('toggle');
+  });
+
+  document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
+    nav.classList.remove('nav-active');
+    burger.classList.remove('toggle');
+  }));
+};
+
+navSlide();
```

Comment:
- [ ] Please use the correct color `#67798E` for the close(X) button.

### Your design
![image](https://user-images.githubusercontent.com/83803414/195098115-1c6f8b9c-608f-47a3-80cd-d81556ddc329.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/195098167-fb3bedc2-0453-4e88-bdd4-a882e67f3955.png)